SYCL/AOT/multiple-devices.cpp

```diff
@@ -10,70 +10,50 @@
 // UNSUPPORTED: cuda
```

Comment:
I assume we can update `REQUIRES` now to exclude a couple of tools from there, right?

Reply:
Good idea, done in https://github.com/intel/llvm-test-suite/pull/603/commits/dd9c661e468f7c612d96297cc44a46acef5fa680